src/app/modules/group/components/pending-request/pending-request.component.ts

```diff
@@ -50,6 +50,7 @@ export class PendingRequestComponent implements OnInit, OnChanges {
     { field: 'user.login', header: 'USER' },
     { field: 'at', header: 'REQUESTED ON' },
   ];
+  groupColumn = { field: 'group.name', header: 'GROUP'};
```

Comment:
These two variables `columns` and `groupColumn` aren't the same. `columns` is an instance variable that may change and has a default value. However, you use `groupColumn` as a constant from what I see. So it should be a constant (moved over the class definition).

Reply:
fixed